README.md

```diff
@@ -309,6 +307,11 @@ $ curl -X POST -H "Authorization: token=$AUTH_TOKEN" http://<dcos_url>/service/c
 4. Once the restore operation is finished, check if the data is restored correctly.
 5. [Uninstall](https://github.com/mesosphere/dcos-cassandra-service#uninstall) old cluster.
 
+**Upgrade Notes:** 
```

Comment:
I think this should be up at line 302. This is more important than the section on the special upgrade path.